### PR TITLE
travis fails if go fmt makes changes

### PR DESCRIPTION
**Pull request by DavidVorick**
I have made a small change to accept.go to demonstrate that travis will fail.

`make` will not fail if the code is not formatted, only `make fmt-strict`, which means workflow should not be disrupted. Travis runs `make fmt-strict` before running make to catch the problem.

See the first commit for a demonstration of the build failing. Second commit fixes this so we don't merge failed code. And additional useful change was made so it's not a wasted commit.

### CODE REVIEW

**Comment by DavidVorick** [comment]
I switched it to go 1.4.1 because travis has go 1.4.1 preinstalled, but not go 1.4 preinstalled. It just makes life easier on travis.

**Comment by DavidVorick** [comment]
Fixes #274

File: Makefile [hunk 1]
```diff
@@ -6,6 +6,9 @@ all: install
 fmt:
 	go fmt ./...
 
+fmt-strict:
+	@exit $(shell go fmt ./... | wc -l)
+
```
Comment (lukechampine) [inline Makefile]: can this be moved to the .travis file? I don't feel like it belongs in the Makefile.

**Comment by lukechampine** [comment]
I think that commit broke Travis lol. You need to run the `exit` in a subshell, otherwise it will terminate the Travis process. Try `$(exit $(go fmt ./... | wc -l))`

**Comment by DavidVorick** [comment]
urg. Looks like `go get` is failing b/c go.googlesource.com is down? :(

**Comment by lukechampine** [comment]
it's working for me locally now. Try again (commit --amend with no changes)

**Pull request merged**